README.md

```diff
@@ -1,57 +1,63 @@
-# QualityMetricsCli
+# Code PushUp CLI
 
-<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
+**Quality metrics for web projects, highly configurable.**
 
-✨ **This workspace has been generated by [Nx, a Smart, fast and extensible build system.](https://nx.dev)** ✨
+This monorepo contains code for open-source Code PushUp NPM packages:
 
-## Generate code
+- [@code-pushup/cli](./packages/cli) - CLI for collecting audit results and uploading report to portal
```

Comment:
Let's add PR comments
```suggestion
- [@code-pushup/cli](./packages/cli) - CLI for collecting audit results and uploading report to portal or add it as comment to your PR
```

Reply:
I wouldn't consider that a primary use case at the moment, so I wouldn't mention it in the CLI summary. I guess you could put the `report.md` into a GitHub comment (if it fits within character limit), but it'll be a lot more useful to only have the diff there (a feature we definitely want at some point, but not in MVP scope).